tests/Unit/EmogrifierTest.php

```diff
@@ -267,6 +267,18 @@ public function emogrifyBodyContentKeepsSpecialCharacters($codeNotToBeChanged)
         static::assertContains($codeNotToBeChanged, $result);
     }
 
+    /**
+     * @test
+     */
+    public function addsMissingHtml5DocumentType()
+    {
+        $this->subject->setHtml('<html></html>');
+
+        $result = $this->subject->emogrifyBodyContent();
```

Comment:
As far as I can see, the only issue (the cause of failed test) is that this line should read `$result = $this->subject->emogrify()`.

Reply:
Oh, of course. D'oh!